Makefile

```diff
@@ -94,7 +94,12 @@ tests:
 	go test ./...
 
 vet:
-	go vet ./...
+	go tool vet --all .
```

Comment:
Ok well nothing except the manual vet is using the go tool invocation.  Could please change it back, sorry.